src/helm/benchmark/scenarios/lextreme_scenario.py

```diff
@@ -0,0 +1,276 @@
+            'I-COUNTRY',
+            'B-PLACE',
+            'I-PLACE',
+            'B-TERRITORY',
+            'I-TERRITORY',
+            'I-UNIT',
+            'B-UNIT',
+            'B-VALUE',
+            'I-VALUE',
+            'B-YEAR',
+            'I-YEAR',
+            'B-STANDARD ABBREVIATION',
+            'I-STANDARD ABBREVIATION',
+            'B-MONTH',
+            'I-MONTH',
+            'B-DAY',
+            'I-DAY',
+            'B-AGE',
+            'I-AGE',
+            'B-ETHNIC CATEGORY',
+            'I-ETHNIC CATEGORY',
+            'B-FAMILY NAME',
+            'I-FAMILY NAME',
+            'B-INITIAL NAME',
+            'I-INITIAL NAME',
+            'B-MARITAL STATUS',
+            'I-MARITAL STATUS',
+            'B-PROFESSION',
+            'I-PROFESSION',
+            'B-ROLE',
+            'I-ROLE',
+            'B-NATIONALITY',
+            'I-NATIONALITY',
+            'B-TITLE',
+            'I-TITLE',
+            'B-URL',
+            'I-URL',
+            'B-TYPE',
+            'I-TYPE',
+        ],
+    }
+
+    def __init__(self, subset: str):
+        assert subset in task_code_mapping.keys(), f"Unknown subset: {subset}"
+        self.subset = subset
+
+    def get_instances_for_subset(self, config: str) -> List[Instance]:
+        task_code = task_code_mapping[config]
+        # Load dataset
+        dataset = load_dataset(self.dataset_name, config)
+
+        if task_code == 'SLTC':
+            class_label = dataset['train'].features["label"]
+            label_classes = class_label.names
+        if task_code in ['MLTC']:
+            # construct the label classes
+            label_classes = set()
+            for split in self.splits_mapping.values():
+                for example in dataset[split]:
+                    label_classes |= set(example["label"])  # add all new labels to the set
+            label_classes = sorted(list(label_classes))
+        if task_code == 'NER':
+            label_classes = self.ner_class_mapping[config]
+
+        def generate_instance(example, split: str):
+            # TODO implement this also for NER and test that it works for MLTC
+            if task_code == 'SLTC':
+                correct_label = class_label.int2str(example['label'])  # get label name for correct label
+                correct_labels = correct_label if isinstance(correct_label, list) else [correct_label]
+            if task_code == 'MLTC':
+                correct_labels = example['label']  # here we don't have any mapping to label names
+            if task_code == 'NER':
+                correct_labels = [label_classes[label] for label in example['label']]
+
+            # construct wrong references
+            wrong_references = []
+            if task_code in ['SLTC', 'MLTC']:
+                for label_name in label_classes:
+                    if label_name not in correct_labels:
+                        wrong_reference = Reference(output=label_name, tags=[])  # Wrong output
```

Comment:
I'm getting:

```
  File "/home/yifanmai/oss/helm/src/helm/benchmark/adapter.py", line 620, in construct_example_prompt
    result += self.adapter_spec.output_prefix + output + self.adapter_spec.output_suffix
TypeError: can only concatenate str (not "int") to str
```

The argument passed to `output` should be a `str`, but apparently `label_name` can be a `int`. One possible fix would be to make this `output=str(label_name)`. The same applies below for `correct_references`.